CONTRIBUTING.md

```diff
@@ -58,6 +58,9 @@ PyTorch, see the [Contributing Guide](docs/source/community/contribution_guide.r
 
 ## Developing PyTorch
 
+A full set of instructions on installing PyTorch from source is here:
```

Comment:
Especially because this is linked to from internal docs, but also for anyone installing pytorch in general, it makes sense to include the full setup instructions right under "Developing Pytorch Header"